docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -305,6 +305,111 @@ document
 
 </PlatformSection>
 
+### Loading Strategies
+
+Because the `feedbackIntegration` is a user-facing integration, we offer two loading strategies that have implications on bundle size.
+
+<Note>
+For most users, we recommend using `feedbackIntegration` in your `Sentry.init` call. This will set up user feedback with good defaults matching the environment.
+</Note>
```

Comment:
```suggestion
<Note>
For most users, we recommend using `feedbackIntegration` in your `Sentry.init` call. This will set up user feedback with good defaults, matching the environment.
</Note>
```